test/e2e-test-application/src/app/project/settings/settings.component.html

```diff
@@ -91,5 +91,21 @@ <h3 class="fd-layout-panel__title">
         to go to <i>project two</i>, where you can try out hasBack functionality
       </div>
     </div>
+    <div class="fd-layout-panel">
```

Comment:
```suggestion
    <div  *ngIf="isModal" class="fd-layout-panel">
```

It would be nice to have the **isModal** conditional here instead, since in the current place it distorts the non-modal settings view: 
![Screenshot 2020-08-25 at 16 09 43](https://user-images.githubusercontent.com/18335489/91186715-ab13df80-e6ef-11ea-91ce-af25a4f93232.png)